src/models/key-utilities.ts

```diff
@@ -91,6 +92,28 @@ export class KeyUtilities {
     return output;
   }
 
+  static CryptoJsWordArrayToUint8Array(wordArray: CryptoJS.lib.WordArray) {
+    const l = wordArray.sigBytes;
+    const words = wordArray.words;
+    const result = new Uint8Array(l);
+    let i = 0 /*dst*/,
+      j = 0; /*src*/
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
```

Comment:
Seems `while(i < l)` makes more sense here?

Reply:
Agree with you. Corrected